pset-iterators.js

```diff
@@ -66,6 +72,12 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter((presidentJames) => {
+  return presidentJames.president.includes("James")
+})
+
+console.log(presidentsNamedJames)
+
```

Comment:
@rlou88 Good job here using the filter method to return all of the presidents whose name includes James. Nice work using the includes method to look up the string to see if James is in each name.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)